snuba/web/rpc/v1/resolvers/R_eap_spans/common/sampling_in_storage_util.py

```diff
@@ -24,6 +24,9 @@
 from snuba.web.query import run_query
 
 T = TypeVar("T", TimeSeriesRequest, TraceItemTableRequest)
+MetricsBackendType: TypeAlias = Callable[
+    [str, Union[int, float], Optional[Dict[str, str]], Optional[str]], None
+]
```

Comment:
you can define a metrics backend for your file like so:

https://github.com/getsentry/snuba/blob/master/snuba/web/db_query.py#L72

and then you don't need to carry the prefix around